### PR TITLE
follow up namespace type extension

### PR DESCRIPTION
**Pull request by pepyakin**
This changeset is a follow up for #115. It does quite some changes:

1. regenerates subxt to accomodate for the runtime changes (those include not only, 115, but also the rejig related to kusama runtime).
2. updates some strings and doc comments
3. updates the namespaces for the demo
4. fixes adapters

Related #12

### CODE REVIEW

**Comment by pepyakin** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #125** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/125?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #126** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/126?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #123** <a href="https://app.graphite.dev/github/pr/thrumdev/blobs/123?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/125?utm_source=stack-comment).

File: demo/rollkit/restart-local.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 DA_BLOCK_HEIGHT=1
-NAMESPACE=01020304
+NAMESPACE=01020304050607080910111213141516
```
Comment (rphmeier) [inline demo/rollkit/restart-local.sh:2]: This should not pass validation checks for v0

File: demo/sovereign/const-rollup-config/src/lib.rs [hunk 2]
```diff
@@ -1,4 +1,4 @@
 /// The namespace used by the rollup to store its data. This is a raw slice of 8 bytes.
 /// The rollup stores its data in the namespace b"sov-test" on Celestia. Which in this case is encoded using the
 /// ascii representation of each character.
-pub const ROLLUP_NAMESPACE_RAW: [u8; 4] = [3, 0, 0, 0];
+pub const ROLLUP_NAMESPACE_RAW: [u8; 16] = 3_u128.to_be_bytes();
```
Comment (rphmeier) [inline demo/sovereign/const-rollup-config/src/lib.rs:4]: FYI comment is really wrong

Comment (pepyakin) [reply demo/sovereign/const-rollup-config/src/lib.rs:4]: Yep, the entire demo is littered with those comments. Should be addressed in one fell swoop.

**Pull request merged**